### PR TITLE
Install Timer service directly instead of adopting

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: src/services.js [hunk 1]
```diff
@@ -610,7 +610,6 @@ export class Services {
    * @return {!./service/timer-impl.Timer}
    */
   static timerFor(window) {
-    // TODO(alabiaga): This will always return the top window's Timer service.
```
Comment (choumx) [inline src/services.js]: Need to change `getService` to `getExistingServiceForDocInEmbedScope`. 

First line of `getService` is `win = getTopWindow(win)`.

Comment (dvoytenko) [reply src/services.js]: Do you mean it works incorrectly now?

Comment (choumx) [reply src/services.js]: Yea, that's what the TODO meant.

Comment (dvoytenko) [reply src/services.js]: I see. Do you want me to change this here? Would be happy to, but that could have deeper ripples...

Comment (dvoytenko) [reply src/services.js]: I reverted this change for now - too many ripple effects to fight through in a preliminary PR.

File: test/unit/test-extensions.js [hunk 2]
```diff
@@ -1003,13 +1009,14 @@ describes.sandboxed('Extensions', {}, () => {
...
       const navigation = Services.navigationForDoc(any);
-      const timer = Services.timerFor(any);
 
       expect(url.constructor.installInEmbedWindow).to.be.called;
       expect(actions.constructor.installInEmbedWindow).to.be.called;
       expect(standardActions.constructor.installInEmbedWindow).to.be.called;
       expect(navigation.constructor.installInEmbedWindow).to.be.called;
-      expect(timer.constructor.installInEmbedWindow).to.be.called;
+
+      // QQQQQ
```
Comment (choumx) [inline test/unit/test-extensions.js]: Remove.

**Pull request merged**